docs/installation.rst

```diff
@@ -336,6 +334,22 @@ the path to the source directory.
                            if the build is on a separate filesystem from the source, as
                            required on some HPC systems.
 
+- Legacy CUDA version of QMCPACK
+
+  ::
+
+    QMC_CUDA      Enable legacy CUDA code path for NVIDIA GPU acceleration (default: OFF)
+    QMC_CUDA2HIP  AMD GPU support by translating CUDA to HIP (default: OFF)
```

Comment:
QMC_CUDA2HIP is not for Legacy CUDA only. We can put all the GPU related option together.

GPU accelleration of QMCPACK
ENABLE_OFFLOAD. For all OpenMP offload code, and point to the section for details.

NVIDIA GPU:
ENABLE_CUDA
QMC_CUDA mention no AFQMC

AMD GPU:
EANBLE_CUDA + QMC_CUDA2HIP legacy CUDA, mention no AFQMC.
QMC_CUDA + QMC_CUDA2HIP legacy CUDA, mention no AFQMC
ENABLE_HIP AFQMC need this.

Reply:
The directions for AMD AOMP compiler don't include ENABLE_CUDA or QMC_CUDA2HIP.   Should they be there?  If we just list the option it's not clear when it should be used.

Reply:
We will need to update the AOMP recipe with ENABLE_CUDA or QMC_CUDA2HIP added. I think that was enabled after the original AOMP recipe was added.